libopenage/renderer/opengl/shader_program.cpp

```diff
@@ -1,20 +1,21 @@
-// Copyright 2013-2019 the openage authors. See copying.md for legal info.
+// Copyright 2013-2023 the openage authors. See copying.md for legal info.
 
 #include "shader_program.h"
 
 #include <algorithm>
 #include <cstdio>
 #include <unordered_set>
 
+#include "../../datastructure/constexpr_map.h"
 #include "../../error/error.h"
 #include "../../log/log.h"
-#include "../../datastructure/constexpr_map.h"
 #include "../../util/opengl.h"
 
-#include "texture.h"
-#include "shader.h"
-#include "geometry.h"
-#include "lookup.h"
+#include "renderer/opengl/context.h"
+#include "renderer/opengl/geometry.h"
+#include "renderer/opengl/lookup.h"
+#include "renderer/opengl/shader.h"
+#include "renderer/opengl/texture.h"
```

Comment:
isn't that just `"context.h"` etc, no need for include search path injection.

Reply:
see https://github.com/SFTtech/openage/pull/1492#discussion_r1090145590